README.md

```diff
@@ -1,2 +1,70 @@
 # browserify-i18n
 Browserify transform for internationalizing tagged strings
+
+
+## Install
+
+```
+$ npm install https://github.com/yola/browserify-i18n.git
+```
+
+## Usage
+
+For standard, non-optimized usage, do:
+
+```javascript
+var browserify = require('browserify');
+var i18n = require('browserify-i18n');
+var opts = {
+  locale: 'en',                         // The locale code to be used
+  localeDirs: ['./path/to/locale/dir/'] // Array of paths to locale .po files
```

Comment:
in this case you're comparing optimized v. non-optimized and global v. local at the same time, which could be confusing to someone who's trying to figure out what this does. i'd suggest illustrating one feature at a time(in this case, optimized v. non-optimized, where the results would be exactly the same but one takes longer). i don't think we need to show that transform can take options, since we didn't write that functionality. but if you want to, i'd just make sure both examples are consistent